starter-core/src/main/java/io/micronaut/starter/feature/problemjson/ProblemJson.java

```diff
@@ -0,0 +1,53 @@
+package io.micronaut.starter.feature.problemjson;
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.Feature;
+
+import javax.inject.Singleton;
+
+@Singleton
+public class ProblemJson implements Feature {
+
```

Comment:
As version of the module is M1 I think it's better to define the feature as "preview"

Reply:
Done